### PR TITLE
Replace the example pages icon with `"@yamada-ui/lucide"`

### LINKED ISSUE #2043: Replace the example pages icon with `@yamada-ui/lucide`
### Subject

examples

### Description

Currently, We are using the icon from `components/media-and-icons` and `@yamada-ui/fontawesome`, but I would like to replace it with the icon from `@yamada-ui/lucide`.

### Are you willing to participate in creating this documentation page and create a pull request?

None

### PR DESCRIPTION
**Pull request by zonoryo17**
Closes #2043

## Description

Replaced the icon on the sample page with `"@yamada-ui/lucide"`.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Comment by hirotomoyamada** [comment]
@zonoryo17 @108yen @fffk10 @Haru-0001 @hirayamahiroto 

Fixed example pages.

File: docs/pages/examples/playground/header/control-button.tsx [hunk 1]
```diff
@@ -23,7 +22,7 @@ export const ControlButton: FC<ControlButtonProps> = memo(
       <Menu {...menuProps}>
         <MenuButton
           as={IconButton}
-          icon={<Icon icon={faEllipsis} fontSize="1.125em" />}
+          icon={<Ellipsis fontSize="1.3em" />}
```
Comment (fffk10) [inline docs/pages/examples/playground/header/control-button.tsx:25]: I don't think it's necessary to change the fontSize.

File: docs/pages/examples/playground/footer/index.tsx [hunk 2]
```diff
@@ -20,9 +19,7 @@ export const Footer: FC<FooterProps> = memo(({ ...rest }) => {
 
       <IconButton
         variant="ghost"
-        icon={
-          <Icon icon={faClockRotateLeft} fontSize="1.125em" color="muted" />
-        }
+        icon={<History fontSize="2xl" color="muted" />}
```
Comment (fffk10) [inline docs/pages/examples/playground/footer/index.tsx:22]: I don't think it's necessary to change the fontSize.

File: docs/pages/examples/dashboard/cards.tsx [hunk 3]
```diff
@@ -107,7 +101,7 @@ const Item: FC<ItemProps> = memo(
 
           <Spacer />
 
-          <Icon as={icon} color="muted" {...iconProps} />
+          <Icon color="muted" size="2xl" {...iconProps} />
```
Comment (fffk10) [inline docs/pages/examples/dashboard/cards.tsx:104]: The size seems to have changed

before
image


after
image

**Review by fffk10 — CHANGES_REQUESTED** [review]
I confirmed it.
Please check.

File: docs/pages/examples/dashboard/cards.tsx (hunk 3 above)
Comment (hirotomoyamada) [reply docs/pages/examples/dashboard/cards.tsx:104]: @fffk10 

That change is intentional. The icon was too small.

File: docs/pages/examples/playground/footer/index.tsx (hunk 2 above)
Comment (hirotomoyamada) [reply docs/pages/examples/playground/footer/index.tsx:22]: @fffk10

That change is intentional. The icon was too small.

File: docs/pages/examples/playground/header/control-button.tsx (hunk 1 above)
Comment (hirotomoyamada) [reply docs/pages/examples/playground/header/control-button.tsx]: @fffk10

That change is intentional. The icon was too small.

Comment (fffk10) [reply docs/pages/examples/playground/header/control-button.tsx]: Understood, I'll solve it.

**Review by fffk10 — APPROVED** [review]

**Review by 108yen — APPROVED** [review]

**Pull request merged**